docs/src/main/sphinx/develop/client-protocol.rst

```diff
@@ -0,0 +1,257 @@
+    - For reporting purposes, this supplies the name of the software
+      that submitted the query.
+  * - ``X-Trino-Catalog``
+    - The catalog to be used when running the query.  Set by response
+      header ``X-Trino-Set-Catalog``.
+  * - ``X-Trino-Schema``
+    - The schema to be used when running the query.  Set by response
+      header ``X-Trino-Set-Schema``.
+  * - ``X-Trino-Time-Zone``
+    - The timezone to be used when running the query, which by default
+      is the timezone of the Trino engine.
+  * - ``X-Trino-Language``
+    - The language to be used when running the query and formatting
+      results.  The language of the session can be set on a
+      per-query basis using the ``X-Trino-Language`` HTTP header,
+      or via the ``TrinoConnection.setLocale(Locale)`` method in
+      the JDBC driver.
+  * - ``X-Trino-Trace-Token``
+    - Supplies a trace token to the Trino engine to help identify
+      log lines that originate with this query request.
+  * - ``X-Trino-Session``
+    - Supplies a comma-separated list of name=value pairs as session
+      properties.  When the Trino client run a
+      ``SET SESSION name=value`` query, the name=value pair
+      is returned in the ``X-Set-Trino-Session`` response header,
+      and added to the client's list of session properties.
+      If the response header ``X-Trino-Clear-Session`` is returned,
+      its value is the name of a session property that will be
+      removed from the client's accumulated list.
+  * - ``X-Trino-Role``
+    - Sets the catalog role that will be used in this request.
+      Set by response header ``X-Trino-Set-Role``.
+  * - ``X-Trino-Prepared-Statement``
+    - A comma-separated list of the name=value pairs, where the
+      names are names of previously prepared SQL statements, and
+      the values are keys that identify the executable form of the
+      named prepared statements.
+  * - ``X-Trino-Transaction-Id``
+    - The transaction ID to be used when running the query.  Set
+      by response header ``X-Trino-Started-Transaction-Id`` and
+      cleared by ``X-Trino-Clear-Transaction-Id``.
+  * - ``X-Trino-Client-Info``
+    - Contains arbitrary information about the client program
+      submitting the query.
+  * - ``X-Trino-Client-Tags``
+    - A comma-separated list of "tag" strings, used to identify
+      Trino resource groups.
+  * - ``X-Trino-Resource-Estimate``
+    - A comma-separated list of ``resource=value`` type
+      assigments.  The possible choices of ``resource`` are
+      "EXECUTION_TIME", "CPU_TIME",  "PEAK_MEMORY" and
+      "PEAK_TASK_MEMORY".  "EXECUTION_TIME" and "CPU_TIME"
+      have values specified as airlift ``Duration`` strings,
+      whose format is a double precision number followed by
+      a ``TimeUnit`` string, e.g., of ``s`` for seconds,
+      ``m`` for minutes, ``h`` for hours, etc.  "PEAK_MEMORY" and
+      "PEAK_TASK_MEMORY" are specified as as airlift ``DataSize`` strings,
```

Comment:
Btw .. we really should have a separate page about these units in admin properties section.

Reply:
But not for this PR, right?

Reply:
yeah ...